content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -25,103 +25,76 @@ is a replacement for the `istio-init` container that performs the same
-       where `ServiceAccounts` are utilized.
+### Prerequisites
 
-## Basic Installation
+1. Install Kubernetes with the container runtime supporting CNI and `kubelet` configured
+  with the main [CNI](https://github.com/containernetworking/cni) plugin enabled via `--network-plugin=cni`.
+    * AWS EKS, Azure AKS, and IBM Cloud IKS clusters have this capability.
+    * Google Cloud GKE clusters has CNI enabled when any of the following features is enabled:
+       [network policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy),
+       [intranode visibility](https://cloud.google.com/kubernetes-engine/docs/how-to/intranode-visibility),
+       [workload identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity),
+       [pod security policy](https://cloud.google.com/kubernetes-engine/docs/how-to/pod-security-policies#overview),
+       or [dataplane v2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2).
+    * OpenShift has CNI enabled by default.
 
-In most environments, a basic Istio cluster with CNI enabled can be installed using the following command:
+1. Install Kubernetes with the [ServiceAccount admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#serviceaccount) enabled.
+    * The Kubernetes documentation highly recommends this for all Kubernetes installations
+      where `ServiceAccounts` are utilized.
 
-{{< text bash >}}
-$ cat <<EOF > istio-cni.yaml
+### Install Istio with CNI plugin
+
+In most environments, a basic Istio cluster with CNI enabled can be installed using the following configuration:
+
+{{< text yaml >}}
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   components:
     cni:
       enabled: true
-  values:
-    cni:
-      excludeNamespaces:
-       - istio-system
-       - kube-system
-      logLevel: info
-EOF
-$ istioctl install -f istio-cni.yaml
 {{< /text >}}
 
-## Advanced Installation
-
-1.  Determine the Kubernetes environment's CNI plugin `--cni-bin-dir` and `--cni-conf-dir` settings.
-    Refer to [Hosted Kubernetes settings](#hosted-kubernetes-settings) for any non-default settings required.
-
-1.  Install Istio CNI and Istio using `istioctl`.
-    Refer to the [Istio install](/docs/setup/install/istioctl/) instructions and pass `--set components.cni.enabled=true` option.
-    Pass `--set values.cni.cniBinDir=...` and/or `--set values.cni.cniConfDir=...` options when installing `istio-cni` if non-default,
-    as determined in the previous step.
+This will deploy an `istio-cni-node` DaemonSet into the cluster, which installs Istio CNI plugin binary to each node and set up needed configuration for the plugin.
+The `istio-cni-node` DaemonSet runs with [`system-node-critical`](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/) `PriorityClass`.
```

Comment:
```suggestion
This will deploy an `istio-cni-node` DaemonSet into the cluster, which installs the Istio CNI plugin binary to each node and sets up the necessary configuration for the plugin.
The CNI DaemonSet runs with [`system-node-critical`](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/) `PriorityClass`.
```